src/makePageElement.ts

```diff
@@ -1,41 +1,61 @@
 import getPageObject from './getPageObject';
 import { fetchRouteData } from './fetchData';
+import type {
+  ExtendedOptions,
+  PageComponents,
+  PageInfo,
+  PageObject,
+} from './commonTypes';
+import { RuntimeEnvironment } from './constants';
 import { renderApp } from './_app';
-import type { ExtendedOptions, Page } from './commonTypes';
 
 /*
- * Return an instance of the page element corresponding
- * to the given path
+ * Return page info associated with a given path
  */
-export default async function makePageElement({
+export async function getPageInfo({
```

Comment:
I don't remember... does it need to be exported?

Reply:
Yes, its used in `getPage`.